### ISSUE TITLE
Create pull request action fails on windows

### ISSUE THREAD

**Issue by deepak1556** [microsoft/vscode-pull-request-github#2046]
Testing #2037

* Start working on issue
* make some commits
* trigger create pull request action from status bar
* Notification arises with `Failed to execute git`

log from the extension

```
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch deepak1556/issue3676
```

**Comment by alexr00** [comment]
@deepak1556, did you see the fork flow when you started working on the issue? Are you using an account that doesn't have permission to push in the repo of the folder you had open? And finally, was the issue you started working on in the repo you had open?

**Comment by deepak1556** [comment]
> did you see the fork flow when you started working on the issue?

nope I didn't see it

> Are you using an account that doesn't have permission to push in the repo of the folder you had open?

yup thats correct

> was the issue you started working on in the repo you had open?

yup its the same repo that i don't have push access to

**Comment by alexr00** [comment]
Can you share any `githubIssues` settings you have set?

**Comment by deepak1556** [comment]
```
"githubIssues.queries": [
        {
            "label": "My Issues",
            "query": "state:open"
        },
        {
            "label": "Created Issues",
            "query": "author:${user} state:open repo:${owner}/${repository} sort:created-desc"
        }
    ],
    "githubPullRequests.remotes": [
        "origin"
    ],
```

Repo used for testing was, 

```
PS D:\github\nsfw> cat .\.git\config
[core]
        repositoryformatversion = 0
        filemode = false
        bare = false
        logallrefupdates = true
        symlinks = false
        ignorecase = true
[remote "origin"]
        url = https://github.com/Axosoft/nsfw
        fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
        remote = origin
        merge = refs/heads/master
```

**Comment by alexr00** [comment]
@deepak1556 I still don't see anything suspicious. Any chance you were still signed in with an account that does have access to Axosoft/nsfw?

**Comment by alexr00** [comment]
Were you using the nightly build of the extension? https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github-insiders

**Comment by deepak1556** [comment]
Sorry for the delayed response, I was not using the nightly version previously, installed it now and I still see the same error. I also tried with another repository which I clearly don't have push access to https://github.com/wolfpld/tracy

I just tried on macOS and got the same error `Failed to execute git`, looking a bit more on the issue column, all the issues populated are from some random repositories. I adjusted the query to 

```
"githubIssues.queries": [
        {
            "label": "My Issues",
            "query": "state:open repo:wolfpld/tracy sort:created-desc"
        }
    ],
```

now the flow for fork works as expected, tested on both windows and macOS. Maybe the error information can be improved when working on issues not related to the repository currently opened in the workspace ?
